submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionStorageModeTest.kt

```diff
@@ -142,11 +151,15 @@ class SubmissionStorageModeTest(
 
     @Test
     fun `10-4 transfer from FIRE to NFS`() {
-        val submission = createSubmission("S-STR-MODE-4")
-
+        val (submission, file, fileList, fileListFile) = createSubmission("S-STR-MODE-4")
+        webClient.uploadFiles(listOf(file, fileListFile, fileList))
         assertThat(webClient.submitSingle(submission, TSV, FIRE)).isSuccessful()
         val fireSub = submissionRepository.getExtByAccNo("S-STR-MODE-4", includeFileListFiles = true)
 
+        // We upload files manually into S3 to simulate fire S3 support.
```

Comment:
Just to clarify, is this to validate the case where the imaging team uploads the files directly to FIRE? If so, I don't think this is the right place to validate this logic since the idea of the transfer function is for submissions that already exist and will be transferred as-is so it seems weird to me we're adding files afterward.

I think the proper test case for this would be [here](https://github.com/EBIBioStudies/biostudies-backend-services/pull/702/files#diff-f291be59aef177e9e2d92fe673f8885390fe05251ff8400078e5b6c9e97d98b8R73) like this:
1. Submit to FIRE
2. Upload extra files manually
3. Resubmit to NFS including the manually uploaded files in the new version

Reply:
No, this is not because of this. This changes are because we need to simulate fire mechanism that make every file uploaded through http it make it avaiable in s3.

Reply:
wonderinf if the comment need to be better to clarify this